packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -287,6 +319,12 @@ const DashboardChartTile: FC<Props> = (props) => {
                             text="Explore from here"
                             href={exploreFromHereUrl}
                         />
+                        {savedQueryWithDashboardFilters && (
```

Comment:
I could also check `savedQueryWithDashboardFilters.chartConfig.type !== TABLE` here, let me know if you have any preference

Reply:
Done